### PR TITLE
Database driven homepage

### PR DESCRIPTION
**Pull request by justinmc**
This PR gets the database driven parts of the homepage working (run and view counts).  Related issue: https://github.com/Autodesk/molecular-design-applications/issues/76

### CODE REVIEW

**Comment by justinmc** [comment]
These runs and views are database driven:

screen shot 2017-01-25 at 5 44 32 pm

These "creator images" are database driven:

screen shot 2017-01-25 at 5 45 02 pm

**Comment by justinmc** [comment]
@dionjwa This needs a quick review whenever you get a chance.  Ignore the css stuff, the main important part is just how I'm calculating runs and views.

File: server/main/version.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ console.log("__dirname=" + __dirname);
 // Path to package.json and VERSION files are same - at root of this node application. 
 // Use the same logic to find both files.
 // In a typcial container, this usually means '/app/server' 
-+var packageJSON = require(__dirname + '/../package.json');
-+var versionFilePath = __dirname + '/../VERSION';
+var packageJSON = require(__dirname + '/../package.json');
+var versionFilePath = __dirname + '/../VERSION';
```
Comment (dionjwa) [inline server/main/version.js]: I'm going to change this again in my own PR. These files are in the working dir, and that doesn't change. So they should just be gotten with fs.getFile* and use the file name. This avoids using `__dirname` which is dependent on the location of the executing module, thus brittle to refactoring. Plus I find `JSON.parse(fs.readFile*` a bit clearer that `require` statements for pure JSON data.

**Comment by justinmc** [comment]
This should include https://github.com/Autodesk/molecular-design-applications/issues/97 instead of calculating pseudo views.

File: server/main/version.js (hunk 1 above)
Comment (justinmc) [inline server/main/version.js]: Sounds good.

**Pull request merged**